setup.cfg

```diff
@@ -1,3 +1,34 @@
+[metadata]
+name = minique
+version: attr:minique.__version__
```

Comment:
`version = attr:minique.__version__` ?

Reply:
Good catch. Wonder how this works at all...

Reply:
:shrug: